<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtClientClass</class>
 <widget class="QWidget" name="QtClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtClientClass</string>
  </property>
  <widget class="QLabel" name="helloUserLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>5</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>439</x>
     <y>22</y>
     <width>191</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="createBankAccountButton">
      <property name="text">
       <string>CREATE BANK ACCOUNT</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="paymentButton">
      <property name="text">
       <string>MAKE A PAYMENT</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="transHistoryButton">
      <property name="text">
       <string>TRANSACTION HSTORY</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="creditButton">
      <property name="text">
       <string>ASK FOR CREDIT</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="deleteBankAccountButton">
      <property name="text">
       <string>DELETE BANK ACCOUNT</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>paymentButton</zorder>
   <zorder>transHistoryButton</zorder>
   <zorder>creditButton</zorder>
   <zorder>deleteBankAccountButton</zorder>
   <zorder>createBankAccountButton</zorder>
  </widget>
  <widget class="QTableView" name="bankAccountTableView">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>41</y>
     <width>381</width>
     <height>251</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshTableButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>300</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../Desktop/icons8-synchronize-26.png</normaloff>../../../../Desktop/icons8-synchronize-26.png</iconset>
   </property>
  </widget>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>helloUserLabel</zorder>
  <zorder>bankAccountTableView</zorder>
  <zorder>refreshTableButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
